typescript/src/generator.ts

```diff
@@ -110,6 +111,14 @@ export class Generator {
       this.bundleConfigs = new BundleConfigClient().fromObject(info);
     }
   }
+  private readIamService() {
```

Comment:
nit: add an empty line before this method please :)

Reply:
done. Thank you!